frontend/www/js/omegaup/components/arena/ClarificationList.vue

```diff
@@ -108,6 +147,46 @@ export default class ArenaClarificationList extends Vue {
     this.$emit('update:activeTab', 'clarifications');
   }
 
+  get assignmentsNames(): string[] {
+    return this.filterAssignments
+      ? [
+          ...new Set(
+            this.clarifications.map((clarification) => {
+              if (
+                'assignment_alias' in clarification &&
+                clarification.assignment_alias
+              ) {
+                return clarification.assignment_alias;
+              }
+              return '';
+            }),
```

Comment:
```suggestion
            this.clarifications
                .map((clarification) => clarification.assignment_alias ?? null),
```

esto funciona? hay que evitar valores mágicos como `''` (habría que ajustar el tipo de retorno a `Array<string | null>`).